# HG changeset patch
# User Thorsten Meyer <thorsten.meyier@gmx.de>
# Date 1223029113 -7200
#      Fri Oct 03 12:18:33 2008 +0200
# Node ID 65c4ac814082240ad73de05e5a4d3ef8e47f5591
# Parent  c9d6071b9696b8ba72496181b5dc64d87fa0f04e
hg syntax bug in mercurial example

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,11 +1,12 @@
 2008-10-03  Thorsten Meyer <thorsten.meyier@gmx.de>
+        * interpreter/contrib.txi: correction of the mercurial example
         * interpreter/container.txi: minor correction of the text
-        
+
 2008-09-25  Søren Hauberg  <hauberg@gmail.com>
 
 	* interpreter/image.txi: Update for imread and imwrite instead of
 	loadimge and saveimage.
 
 2008-09-24  Søren Hauberg  <hauberg@gmail.com>
 
 	* interpreter/image.txi: Document imfinfo.
diff --git a/doc/interpreter/contrib.txi b/doc/interpreter/contrib.txi
--- a/doc/interpreter/contrib.txi
+++ b/doc/interpreter/contrib.txi
@@ -49,17 +49,17 @@ Mercurial.  A good place to start is
 @url{http://www.selenic.com/mercurial/wiki/index.cgi/Tutorial}.
 A simplified contribution sequence could look like this:
 
 @example
 hg clone http://www.octave.org/hg/octave
 cd octave
 # change some sources...
 hg commit -m "make Octave the coolest software ever"
-hg export ../cool.diff
+hg export -o ../cool.diff tip
 # send ../cool.diff via email
 @end example
 
 @node General Guidelines
 @section General Guidelines
 
 All Octave's sources are distributed under the General Public License (GPL).
 Currently, Octave uses GPL version 3.  For details about this license, see
